apps/docs/src/app/core/component-docs/button/examples/button-icons-example.component.html

```diff
@@ -1,15 +1,15 @@
-<button fd-button label="Add to Cart" [glyph]="'cart'"></button>
-<button fd-button label="Add to Cart" [fdType]="'emphasized'" [glyph]="'cart'"></button>
-<button fd-button label="Add to Cart" [fdType]="'ghost'" [glyph]="'cart'"></button>
-<button fd-button label="Filter" [fdType]="'positive'" [glyph]="'filter'"></button>
-<button fd-button label="Filter" [fdType]="'negative'" [glyph]="'filter'"></button>
-<button fd-button label="Filter" [fdType]="'attention'" [glyph]="'filter'"></button>
-<button fd-button label="Filter" [fdType]="'transparent'" [glyph]="'filter'"></button>
+<button fd-button label="Add to Cart"  glyph="cart" ariaLabel="Add to Cart"></button>
```

Comment:
those updates of the doc where `[glyph]` is replaced with `glyph` should be done in separate PR